beacon-chain/blockchain/log.go

```diff
@@ -24,19 +25,20 @@ func logStateTransitionData(b *ethpb.BeaconBlock) {
 	}).Info("Finished applying state transition")
 }
 
-func logBlockSyncStatus(block *ethpb.BeaconBlock, blockRoot [32]byte, finalized *ethpb.Checkpoint, genesisTime uint64) error {
+func logBlockSyncStatus(block *ethpb.BeaconBlock, blockRoot [32]byte, finalized *ethpb.Checkpoint, receivedTime time.Time, genesisTime uint64) error {
 	startTime, err := helpers.SlotToTime(genesisTime, block.Slot)
 	if err != nil {
 		return err
 	}
 	log.WithFields(logrus.Fields{
-		"slot":               block.Slot,
-		"slotInEpoch":        block.Slot % params.BeaconConfig().SlotsPerEpoch,
-		"block":              fmt.Sprintf("0x%s...", hex.EncodeToString(blockRoot[:])[:8]),
-		"epoch":              helpers.SlotToEpoch(block.Slot),
-		"finalizedEpoch":     finalized.Epoch,
-		"finalizedRoot":      fmt.Sprintf("0x%s...", hex.EncodeToString(finalized.Root)[:8]),
-		"timeSinceSlotStart": timeutils.Now().Sub(startTime),
+		"slot":                   block.Slot,
+		"slotInEpoch":            block.Slot % params.BeaconConfig().SlotsPerEpoch,
+		"block":                  fmt.Sprintf("0x%s...", hex.EncodeToString(blockRoot[:])[:8]),
+		"epoch":                  helpers.SlotToEpoch(block.Slot),
+		"finalizedEpoch":         finalized.Epoch,
+		"finalizedRoot":          fmt.Sprintf("0x%s...", hex.EncodeToString(finalized.Root)[:8]),
+		"timeSinceSlotStart":     timeutils.Now().Sub(startTime),
+		"timeSinceBlockReceived": timeutils.Now().Sub(receivedTime),
```

Comment:
`timeSinceBlockReceived` implies the time when beacon node receives the block and this is not the case. This implementation only implies when blockchain service receives the block. We should make it more specific like:

`chainServiceProcessTime`
`timeSinceChainServiceReceived`

We probably can rename the other to ``sinceSlotStartTime` to resonate better

Reply:
Agreed, we were just chatting about this with @nisdas today.